### ISSUE TITLE
problem1

### ISSUE THREAD

**Issue by phanisurya** [tyrian5959/problems-i-solved-in-various-code-challenges#1]
TARS says to Cooper, "For a change, let's have only '#' and '*' for the representation of numbers. # corresponds to 0 and * corresponds to 1. I know it's a stupid thing but I like to make things more fun (or not)."

Cooper doesn't have time for these silly things. So help him in this riduculous task that TARS has given him. Given a number N, just find the Nth number in the representation as described by TARS.

In short,

1st number is *

2nd number is *#

3rd number is ** and so on...

Input Format

The first line has an integer T, the number of test cases.

Then T lines follow that, each line having an integer N.

Constraints

1 <= T <= 1000 1 <= N <= 10 ^ 6

Output Format

For each test case, print the required answer.

Sample Input 0          2     1     2
--
 Sample Output 0          *     *#
